cpu/samd5x/cpu.c

```diff
@@ -152,6 +173,10 @@ void cpu_init(void)
     /* initialize the Cortex-M core */
     cortexm_init();
 
+    /* select buck voltage regulator */
+    SUPC->VREG.bit.SEL = 1;
+    while (!SUPC->STATUS.bit.VREGRDY) {}
```

Comment:
This must be excluded for the SAMR30. As already reported, the `SEL` bit is not defined for this CPU.

Reply:
With your fixup, `tests/periph_pm` is now working like a charm :)